learn/what_is_meilisearch/telemetry.md

```diff
@@ -138,13 +138,18 @@ This list is liable to change with every new version of Meilisearch. It's not be
 | `requests.total_succeeded`              | Total number of successful search requests                                                  | 3456
 | `requests.total_failed`                 | Total number of failed search requests                                                      | 24
 | `requests.total_received`               | Total number of received search requests                                                    | 3480
-| `sort.with_geoPoint`                    | `true` if the sort rule `_geoPoint` was used, otherwise `false`                             | true
+| `sort.with_geoPoint`                    | `true` if the sort rule `_geoPoint` was specified, otherwise `false`                        | true
 | `sort.avg_criteria_number`              | Average number of sort criteria among all search requests containing the `sort` parameter   | 2
-| `filter.with_geoRadius`                 | `true` if the filter rule `_geoRadius` was used, otherwise `false`                          | false
+| `filter.with_geoRadius`                 | `true` if the filter rule `_geoRadius` was specified, otherwise `false`                     | false
 | `filter.most_used_syntax`               | Most used filter syntax among all search requests containing the `filter` parameter         | string
 | `q.max_terms_number`                    | Highest number of terms given for the `q` parameter                                         | 5
 | `pagination.max_limit`                  | Highest value given for the `limit` parameter                                               | 60
 | `pagination.max_offset`                 | Highest value given for the `offset` parameter                                              | 1000
+| `formatting.highlight_pre_tag`          | `true` if `highlightPreTag` was specified, otherwise `false`                                | false
+| `formatting.highlight_post_tag`         | `true` if `highlightPostTag` was specified, otherwise `false`                               | false
```

Comment:
```suggestion
| `formatting.highlight_post_tag`         | `true` if `highlightPostTag` is specified, otherwise `false`                               | false
```